policyengine_us/variables/gov/local/ca/la/dwp/ez_save/la_ez_save.py

```diff
@@ -0,0 +1,15 @@
+from policyengine_us.model_api import *
+
+
+class la_ez_save(Variable):
+    value_type = float
+    entity = SPMUnit
+    definition_period = YEAR
+    label = "Los Angeles County EZ Save program"
+    defined_for = "la_ez_save_eligible"
+
+    def formula(spm_unit, period, parameters):
+        electricity_expense = spm_unit("electricity_expense", period)
+        p = parameters(period).gov.local.ca.la.dwp.ez_save
+        uncapped_amount = p.amount * p.applicable_months
```

Comment:
It's a monthly benefit, shouldn't we multiply by 12

Reply:
I see, I was assuming that you could only get one bill amount per year. But I misread the conversation on X